!!python/object/new:dynamic_reconfigure.encoding.Config
dictitems:
  BA_MAX_STEP: 10
  BLUR_SZ: 3
  BLUR_VAR: 0.5
  DEBUG_FEATURE: false
  FAST_THRES: 20
  FEATURE_OF_PYMD: 2
  INIT_SCALE: 0.1
  KP_BLOCKS: 10
  LOOP_CLOSURE: false
  MAX_FEATURES: 120
  MAX_REPROJ_DIST: 2
  MIN_TRACK_DIST: 0.1
  MIN_TRACK_POINTS: 50
  MONO_INLIER_THRES: 0.7
  PROP_POSE_ITER: 50
  PROP_POSE_PYMD: 2
  REFINE_PIXEL: false
  SCALE_MAX_STEP: 20
  SCALE_PYMD: 1
  TEST_STEREO: false
  groups: !!python/object/new:dynamic_reconfigure.encoding.Config
    dictitems:
      BA_MAX_STEP: 10
      BLUR_SZ: 3
      BLUR_VAR: 0.5
      DEBUG_FEATURE: false
      FAST_THRES: 20
      FEATURE_OF_PYMD: 2
      INIT_SCALE: 0.1
      KP_BLOCKS: 10
      LOOP_CLOSURE: false
      MAX_FEATURES: 120
      MAX_REPROJ_DIST: 2
      MIN_TRACK_DIST: 0.1
      MIN_TRACK_POINTS: 50
      MONO_INLIER_THRES: 0.7
      PROP_POSE_ITER: 50
      PROP_POSE_PYMD: 2
      REFINE_PIXEL: false
      SCALE_MAX_STEP: 20
      SCALE_PYMD: 1
      TEST_STEREO: false
      groups: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      id: 0
      name: Default
      parameters: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      parent: 0
      state: true
      type: ''
    state: []
state: []
